test/correctness/simd_op_check.h

```diff
@@ -0,0 +1,301 @@
+#ifndef SIMD_OP_CHECK_H
+#define SIMD_OP_CHECK_H
+
+#include "Halide.h"
+#include "test/common/halide_test_dirs.h"
+#include "sys/time.h"
```

Comment:
(1) Should be <sys/time.h>
(2) Isn't available on Win32 systems. What do you need from this file?

Reply:
Yikes! remnants from an older version of this branch. I don't need this. Will remove.